common/src/main/java/com/tc/net/protocol/transport/ClientConnectionEstablisher.java

```diff
@@ -608,4 +621,11 @@ public String toString() {
     }
 
   }
+
+  @Override
+  public void onError(ConnectionInfo connInfo, Exception e) {
+    if(this.errorListener!=null){
```

Comment:
lets not change formatting style, use proper spaces etc everywhere in this PR

Reply:
I will take care of this